src/Microsoft.Health.Fhir.SqlServer/Microsoft.Health.Fhir.SqlServer.csproj

```diff
@@ -10,11 +10,11 @@
     <EmbeddedResource Include="Features\Schema\Migrations\5.diff.sql" />
     <EmbeddedResource Include="Features\Schema\Migrations\5.sql" />
     <EmbeddedResource Include="Features\Schema\Migrations\6.diff.sql" />
-    <EmbeddedResource Include="Features\Schema\Migrations\6.sql">
-      <InputToImmutableSqlGenerator>true</InputToImmutableSqlGenerator>
-    </EmbeddedResource>
+    <EmbeddedResource Include="Features\Schema\Migrations\6.sql" />
```

Comment:
We need to keep this around for backwards compat.
Actually, we are in a broken state right now, and it's my doing. The code is supposed to be backwards compatible with schema version 4, but that is not the case right now.